src/config_trapdoor.c

```diff
@@ -984,6 +987,21 @@ TbBool parse_trapdoor_trap_blocks(char *buf, long len, const char *config_textna
                   COMMAND_TEXT(cmd_num), block_buf, config_textname);
           }
           break;
+      case 38: // PLACESOUND
+          if (get_conf_parameter_single(buf, &pos, len, word_buf, sizeof(word_buf)) > 0)
+          {
+              n = atoi(word_buf);
+              if ( (!SoundDisabled) && ( (n < 0) || (n > (samples_in_bank - 1)) ) )
```

Comment:
The configuration of traps is not dependent on if the player has his sounds enabled or not I would think.

Say I start a map with my sound disabled, then save. Then next time I load the level (which loads the configs) but with sound enabled, then I want the correct placement sound.
SoundDisabled is checked when playing sounds.

Reply:
yes, config_objects.c:555 does not seem correct either.

Reply:
Do you prefer a new PR for config_objects.c or should I include that in this PR?

Reply:
sorry, never saw you ask the question. I would have answered you could have done it here, but it seems this gave issues.